### PR TITLE
B 21432 display market code int

### PR DESCRIPTION
**Pull request by TevinAdams**
## Summary

In this PR we are displaying the market code on the shipment card headers in MyMove and Office.

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

## MyMove
1. Access mymove as a customer
2. Create a move or use an existing one
3. Create one or many shipments of various types
4. You should see the market code next to the shipment type on the she shipment cards
Screenshot 2024-10-16 at 9 07 37 AM
image

## Office
1. Access office as any role but Prime
2. Create a move or use an existing one
3. Create one or many shipments of various types
4. You should see the market code next to the shipment type on the she shipment cards

image
image

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

## 508 Checks
image
image
image

### CODE REVIEW

File: src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx [hunk 1]
```diff
@@ -205,6 +218,11 @@ describe('HHGShipmentCard component', () => {
     // verify onclick is getting json string as parameter
     expect(mockedOnIncompleteClickFunction).toHaveBeenCalledWith('HHG 1', 'ABC123K-01', 'HHG');
   });
+
+  it('renders incomplete HHGShipmentCard with a heading that as a market code and shipment type', async () => {
```
Comment (traskowskycaci) [inline src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx:222]: small typo here - as should be has

File: src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ const defaultProps = {
   shipmentNumber: 1,
   shipmentId: '#ABC123K',
   shipmentLocator: '#ABC123K-01',
+  marketCode: 'd',
```
Comment (traskowskycaci) [inline src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx:20]: I see lots of good test cases for marketCode: 'd' - would you mind adding in a few for marketCode: 'i'?

File: src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx [hunk 3]
```diff
@@ -191,6 +199,11 @@ describe('HHGShipmentCard component', () => {
     expect(screen.queryByText('Incomplete')).toBeNull();
   });
 
+  it('renders complete HHGShipmentCard with a heading that as a market code and shipment type', async () => {
```
Comment (traskowskycaci) [inline src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx:202]: small typo here - as should be has

File: src/components/Customer/Review/Summary/Summary.jsx [hunk 4]
```diff
@@ -306,6 +310,7 @@ export class Summary extends Component {
             showEditAndDeleteBtn={showEditAndDeleteBtn}
             status={shipment.status}
             onIncompleteClick={this.toggleIncompleteShipmentModal}
+            marketCode={shipment.marketCode}
           />
         );
       }
```
Comment (traskowskycaci) [inline src/components/Customer/Review/Summary/Summary.jsx:316]: Not seeing a market_code for a UB on the summary page - looks like we need a `marketCode={shipment.marketCode}` addition for line 338 inside this `shipment.shipmentType === SHIPMENT_TYPES.UNACCOMPANIED_BAGGAGE` if condition

image

File: src/components/Office/ShipmentHeading/shipmentHeading.module.scss [hunk 5]
```diff
@@ -54,4 +54,10 @@
       color: #ffffff;
     }
   }
+
+  .marketCodeIndicator {
+    font-weight: 100;
+    font-size: 14px;
```
Comment (traskowskycaci) [inline src/components/Office/ShipmentHeading/shipmentHeading.module.scss:60]: What do you think of making this font-size slightly bigger? As is, as an office user viewing it on the MTO page, I personally think the market code looks slightly smaller than it should, proportionally.

As is - 14px:
image

Example of upping that to 20px:
image

**Review by traskowskycaci — CHANGES_REQUESTED** [review]

File: src/components/Office/ShipmentHeading/shipmentHeading.module.scss (hunk 5 above)
Comment (TevinAdams) [reply src/components/Office/ShipmentHeading/shipmentHeading.module.scss]: I agree that I do like them with a bigger font. I increased all of the font for these to at least 16. then some 20.

File: src/components/Customer/Review/Summary/Summary.jsx (hunk 4 above)
Comment (TevinAdams) [reply src/components/Customer/Review/Summary/Summary.jsx:316]: Ahhh this is in int but not main so I missed this one. Its added now. Good catch.

File: src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx (hunk 3 above)
Comment (TevinAdams) [reply src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx]: Went through and fixed the typo on this text and the other ones I found.

File: src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx (hunk 2 above)
Comment (TevinAdams) [reply src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx]: I went through and switched a few of them to use "i".

File: src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx (hunk 1 above)
Comment (TevinAdams) [reply src/components/Customer/Review/ShipmentCard/HHGShipmentCard/HHGShipmentCard.test.jsx]: Went through and fixed the typo on this text and the other ones I found.

File: src/components/Customer/Review/Summary/Summary.jsx (hunk 4 above)
Comment (traskowskycaci) [reply src/components/Customer/Review/Summary/Summary.jsx:316]: Ah yes, totally understand on that one. Hopefully getting the first wave of UB work PO accepted this week

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]
Could you also update the prime sim? Doesn't have to be anything fancy, but was thinking here:
Screenshot 2024-10-21 at 9 23 57 AM

**Comment by TevinAdams** [comment]
> Could you also update the prime sim? Doesn't have to be anything fancy, but was thinking here: Screenshot 2024-10-21 at 9 23 57 AM

@danieljordan-caci  added.
image

**Review by danieljordan-caci — APPROVED** [review]
LGTM!

Screenshot 2024-10-21 at 10 18 46 AM

**Review by traskowskycaci — APPROVED** [review]
Looks good!

**Comment by danieljordan-caci** [comment]
You do have some client tests failing that aren't flaky - just saw

**Pull request merged**